packages/api/tests/endpoints/test-reconciliation-reports.js

```diff
@@ -228,20 +269,75 @@ test.serial('delete a report', (t) =>
       .set('Accept', 'application/json')
       .set('Authorization', `Bearer ${jwtAuthToken}`)
       .expect(200);
-    t.deepEqual(response.body, { message: 'Report deleted' });
+    t.deepEqual(response.body, {
```

Comment:
This seems to be deleting a report that was created in `before`, not `beforeEach`. If that's the case, there is a real strong possibility that these tests can interfere with each other.

Reply:
The tests are run in serial and need a whole refactor which seems out of scope.